xbmc/pvr/guilib/PVRGUIActions.cpp

```diff
@@ -1218,12 +1218,29 @@ namespace PVR
     {
       CPVRStreamProperties props;
 
+      // Need a better way of telling if we're currently playing a live stream
+      //if (item->IsPVRChannelWithArchive() || item->IsEPGWithArchive())
+      client->CloseLiveStream();
```

Comment:
???

Reply:
It's essentially a hack to tell the addon it should reset any offsets for the current streaming session.

Reply:
We wouldn't need this if we have an inputstream addon.